src/main/scala/org/renci/cam/QueryService.scala

```diff
@@ -56,33 +74,31 @@ object QueryService extends LazyLogging {
     edge: TRAPIQueryEdge): RIO[ZConfig[AppConfig] with HttpClient, (Set[String], Set[(String, String)], String)] =
     for {
       edgeType <- ZIO.fromOption(edge.`type`).orElseFail(new Exception("failed to get edge type"))
-      queryText = s"""SELECT DISTINCT ?predicate WHERE {
-           |?predicate <http://www.w3.org/2000/01/rdf-schema#subPropertyOf>+ <${edgeType.iri.value}> .
-           |FILTER NOT EXISTS { ?predicate a <https://w3id.org/biolink/biolinkml/meta/types/SlotDefinition> }
-           |}""".stripMargin
-      query <- Task.effect(QueryFactory.create(queryText))
-      resultSet <- SPARQLQueryExecutor.runSelectQuery(query)
+      queryText =
+        sparql"""SELECT DISTINCT ?predicate WHERE { ?predicate $rdfSchemaSubPropertyOf+ ${edgeType.iri} . FILTER NOT EXISTS { ?predicate a $biolinkmlSlotDefinition } }"""
+      resultSet <- SPARQLQueryExecutor.runSelectQuery(queryText.toQuery)
       predicates = (for {
-          solution <- resultSet.asScala
-          v <- solution.varNames.asScala
-          node = solution.get(v)
-        } yield s"<$node>").mkString(" ")
+        solution <- resultSet.asScala
+        v <- solution.varNames.asScala
+        node = solution.get(v)
+      } yield s"<$node>").mkString(" ")
```

Comment:
Is this just an alignment change? Otherwise I guess this part isn't using the sparql interpolator yet.